src/services/token.services.js

```diff
@@ -0,0 +1,30 @@
+const { Token } = require('../models/token');
+
+const getToken = async (userId) => {
+  const token = await Token.findOne({ where: { userId } });
+
+  return token;
+};
+
+const saveToken = async (userId, refreshToken) => {
+  const token = await getToken(userId);
+
+  if (!token) {
+    return Token.create({ userId, refreshToken });
```

Comment:
Ensure that the `Token.create` method includes all necessary fields, such as `expireAt`, if applicable. This will ensure that the token is fully initialized when created.